.travis.yml

```diff
@@ -38,4 +38,4 @@ script:
   - py.test
   # test that the py2cyto conda environment can be created as well
   - source deactivate
-  - conda env create -f visualization_scripts/minimal_env.yml
+  - conda env create -f workflow/minimal_env.yml
```

Comment:
If we want to run new tests for the workflow, we can add new lines after this one to do the following:
- activate the new environment
- run the workflow code to generate the annotations file
- run pytest on the test case in the `workflow/` subdirectory